### ISSUE TITLE
build with modern VTK 

### ISSUE THREAD

**Issue by amigo421** [jakobwilm/slstudio#20]
hello, 

this is not a bug but just a assistance request.
I've tried to build the project , 
for that , I've cloned VTK (v8) but seems you guys are using another version because libs are absolutely different from what you are referring to in your PRO file.
have you tried to build with latest VTK?

**Comment by jakobwilm** [comment]
Hello.

We use vtk6 from Ubuntu 16.04 repos (see readme). We have not tried with anything else.

/Jakob

On 15 Dec 2017, at 18:10, Ernst Maurer <notifications@github.com<mailto:notifications@github.com>> wrote:


hello,

this is not a bug but just a assistance request.
I've tried to build the project ,
for that , I've cloned VTK (v8) but seems you guys are using another version because libs are absolutely different from what you are referring to in your PRO file.
have you tried to build with latest VTK?
